docs/specs/git.yml

```diff
@@ -75,10 +75,10 @@ inputs:
   user_profile.json: |
     {
       "users": [
-        {"login": "invalid_user"},
-        {"name": "Osmond Jiang", "login": "OsmondJiang", "id": 19990166, "emails": ["xinjiang@microsoft.com"]},
-        {"name": "Renze Yu", "login": "superyyrrzz", "id": 3831744, "emails": ["renzeyu@microsoft.com","superyyrrzz@gmail.com"]},
-        {"name": "Yufei Huang", "login": "yufeih", "id": 511355, "emails": ["yufeih@users.noreply.github.com","yufeih@live.com"]}
+        {"login": "invalid_user", "expiry":"2099-01-01T00:00:00.00000Z"},
```

Comment:
can the system generate a new `expiry` if it equals _default_?

Reply:
We have set Expiry to nullable, it will never expire when Expiry is null. Think we can just remove them here.

---
In reply to: [222885677](https://github.com/dotnet/docfx/pull/3467#discussion_r222885677) [](ancestors = 222885677)

Reply:
agree

---
In reply to: [223545860](https://github.com/dotnet/docfx/pull/3467#discussion_r223545860) [](ancestors = 223545860,222885677)